src/literature/components/PublicationInfo.jsx

```diff
@@ -3,9 +3,41 @@ import PropTypes from 'prop-types';
 import { Map } from 'immutable';
 
 class PublicationInfo extends Component {
+  displayArtidPageInfoAndJournalIssue() {
```

Comment:
Seems like too much duplication, so it can be written like this:

```js
if (info.has('page_start') && info.has('page_end')) {
  return `${info.get('page_start')}-${info.get('page_end')}`
} else if (info.has('page_start')) {
  return info.get('page_start') 
} else if (info.has('artid')) {
  return info.get('artid');
}
return null;
```

Reply:
`getPageOrArtidDisplay`